### PR TITLE
[YLD-68][Update design layout]

### PR DESCRIPTION
**Pull request by desamtralized**

### CODE REVIEW

File: app/app/blurredCircle.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+"use client";
```
Comment (proofmancer) [inline app/app/blurredCircle.tsx:1]: If it's the same as in the landing page, we should put in the shared later

File: app/app/navbar.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
+"use client";
+
+import Image from "next/image";
+import WalletButton from "@/app/walletButton";
+
+export default function Navbar() {
+  return (
+    <nav className="flex w-auto h-20 px-5 pl-10 py-5 
+      justify-between items-center flex-shrink-0 rounded-3xl border-4 border-[#FFF] 
+      bg-[radial-gradient(152.17%_50%_at_50%_50%,rgba(255,255,255,0.20)_0%,rgba(255,255,255,0.80)_100%)] mx-32 mb-9">
```
Comment (proofmancer) [inline app/app/navbar.tsx:10]: we should create a custom class for this radial

File: app/app/walletButton.tsx [hunk 3]
```diff
@@ -29,10 +29,11 @@ export default function WalletButton() {
   return (
     <>
       <a
-        className="font-mono uppercase tracking-widest p-4 border-r text-sm leading-6 bg-primary cursor-pointer"
+    className="flex w-48 px-7 py-3 justify-center items-center gap-2.5 flex-shrink-0 rounded-full bg-[#007D98]
+        text-white text-center font-sans text-sm font-bold leading-5"
         onClick={onClick}
       >
-        {connectedWallets > 0 ? "WALLETS" : "CONNECT WALLET"}
+        {connectedWallets > 0 ? "Wallets" : "Connect Wallet"}
```
Comment (proofmancer) [inline app/app/walletButton.tsx:36]: What do we want to show here precisely?  We can have one wallet connected at a time, we can have multiple previous connected wallets, that wont ask for permission, but here maybe we can show the first 4 and last 4 chars of the wallet if it's connected

**Review by proofmancer — CHANGES_REQUESTED** [review]
Added some comments, for the wallet and styling

File: app/app/walletButton.tsx (hunk 3 above)
Comment (desamtralized) [reply app/app/walletButton.tsx:36]: We can have many wallets connected at the same time, for now it supports multi connection for BTC and ETH wallets.

**Review by proofmancer — APPROVED** [review]

**Pull request merged**